### DISCUSSION TITLE
Can I Export metadata of Firestorage files?

### DISCUSSION THREAD

**Discussion by KKimj** [AntlerVC/firetable#728] [Support / Q&A] [answered]
I want to get Download-urls all of Firestorage files easily!

Can I do this with rowy?

Thanks!!

**Comment by harinij** [comment, marked as answer]
Hi @KKimj if you have the Images/Files in a column, then you can have another derivative column that lists the URL of the firestore location from the metadata json. The details of how you can achieve this is answered here: https://github.com/rowyio/rowy/discussions/724#discussioncomment-3194162 

Hope this solves your usecase.
